src/renderer/components/settings/TrayColorPicker.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0
+ * Vesktop, a desktop app aiming to give you a snappier Discord Experience
+ * Copyright (c) 2023 Vendicated and Vencord contributors
+ */
+
+import "./traySetting.css";
+
+import { Margins } from "@vencord/types/utils";
+import { findByCodeLazy } from "@vencord/types/webpack";
+import { Forms } from "@vencord/types/webpack/common";
+import { setCurrentState } from "renderer/patches/tray";
+
+import { SettingsComponent } from "./Settings";
+
+const ColorPicker = findByCodeLazy(".Messages.USER_SETTINGS_PROFILE_COLOR_SELECT_COLOR", ".BACKGROUND_PRIMARY)");
+
+const presets = [
+    "#3DB77F", // discord default ~
+    "#F6BFAC" // Vesktop inpired
+];
+
+export const trayIconPicker: SettingsComponent = ({ settings }) => {
+    if (!settings.tray) return null; // how to disable instead of hiding?
+    return (
+        <div className="tray-settings">
+            <div className="tray-container">
+                <div className="tray-settings-labels">
```

Comment:
we prefix our classes to prevent potential conflicts

vencord usually uses `.vc-*` whereas vesktop usually uses `.vcd-`